.travis.yml

```diff
@@ -1,18 +1,10 @@
 ---
-# Verify this with: http://lint.travis-ci.org/
 language: ruby
 sudo: false
 # Delete dependency locks for matrix builds.
 before_install: rm Gemfile.lock || true
```

Comment:
Agree with @deanwilson that removing the `Gemfile.lock` from the repo is the way to go, in which case this can also go